rasa/core/policies/ensemble.py

```diff
@@ -53,7 +54,7 @@ def __init__(
         self._check_for_important_policies()
 
     def _check_for_important_policies(self):
-        if "MappingPolicy" not in self.policies:
+        if not any([isinstance(policy, MappingPolicy) for policy in self.policies]):
```

Comment:
I think you should be able to get rid of the list `[]` around that statement, it should work fine without

Reply:
It's kinda weird, but it does

Reply:
i guess it's just a generator instead of a generator casted into a list -- good catch akela